src/test/java/RestAssuredTests/TestGet.java

```diff
@@ -0,0 +1,45 @@
+package RestAssuredTests;
+
+import APICalls.GetCall;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import utility.PropertyManager;
+
+import static org.hamcrest.Matchers.equalTo;
+
+
+import java.io.IOException;
+
+public class TestGet extends GetCall {
+    public TestGet() throws IOException {
+    }
+
+
+    static RequestSpecification httpRequest = RestAssured.given().when().header("Content-Type", "application/json");
```

Comment:
it's unusual to have HTTP requests and responses as static fields in a test class, this can lead to issues with test independence and parallel execution
consider moving these into non-static fields or local variables within test methods